app/code/Magento/InventorySales/Plugin/CatalogInventory/StockManagement/ProcessRegisterProductsSalePlugin.php

```diff
@@ -99,20 +99,15 @@ public function aroundRegisterProductsSale(StockManagement $subject, callable $p
         if (empty($items)) {
             return [];
         }
+        $productSkus = $this->getSkusByProductIds->execute(array_keys($items));
+        list($items, $productSkus) = $this->excludeUnsupportedTypes($items, $productSkus);
```

Comment:
What will happen with excluded products? Looks like we need to call origin behaviour for excluded skus

Reply:
Initially I did, but @maghamed in this [comment](https://github.com/magento-engcom/msi/pull/805#discussion_r178241624) said. that we should not call original behavior, so I removed that call.